common/onepanel/base/deploy-core-api.yaml

```diff
@@ -0,0 +1,47 @@
+# onepanel-core deployment
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: onepanel-core
+  name: onepanel-core
+  namespace: onepanel
+spec:
+  template:
+    metadata:
+      name: onepanel-core
+      labels:
+        app: onepanel-core
+    spec:
+      serviceAccountName: onepanel
+      containers:
+        - name: onepanel-core
+          image: onepanel/core:1.3
```

Comment:
let’s use `0.5` for this tag.

Reply:
Sure thing

Reply:
@aleksandrmelnikov thought about this for a while, and `1.0.0-beta.1` is more appropriate. Let's switch to `1.0.0-beta.1` and not increment it while we test and only increment when we release.

Reply:
Made change to deployment files. Uploaded relevant tags to docker hub.